indy-vdr-proxy/src/main.rs

```diff
@@ -48,8 +54,49 @@ pub struct AppState {
     transactions: PoolTransactions,
 }
 
+#[cfg(feature = "fetch")]
+async fn fetch_transactions(genesis: String) -> VdrResult<PoolTransactions> {
+    let https = HttpsConnector::new();
+    let client = Client::builder().build::<_, hyper::Body>(https);
+    let mut res = client
+        .get(
+            genesis
+                .parse()
+                .with_err_msg(VdrErrorKind::Config, "Error parsing genesis URL")?,
```

Comment:
Could put the genesis URL itself into the error message, so the troubleshooter can identify what's wrong with it? (e.g.,
```
.with_err_msg(VdrErrorKind::Config, format!("Error parsing genesis URL {}", genesis.as_str())
```